### PR TITLE
First page documentation

### PR DESCRIPTION
**Pull request by K-Jo**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+
+cd deps
+git clone git://github.com/antirez/redis.git --branch 5.0
+cd redis
+make malloc=libc -j4
+cd ../..
+
+mkdir build
+cd build
+cmake -DDEPS_PATH=../deps/install ..
```
Comment (mnunberg) [inline docs/index.md]: -DDEPS_PATH=../deps-linux-gnu or -DDEPS_PATH=deps-darwin (use `ls` to see the actual directory name).

**Comment by K-Jo** [comment]
Please review https://github.com/RedisAI/RedisAI/blob/e9cdb79ea3786fa7205d8d55d467ff993da8f07b/docs/index.md#start

File: docs/index.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+
+## Quickstart
+
+1. [Docker](#docker)
+2. [Build](#building)
+3. [Start](#start)
+
+## Docker
+
+To quickly tryout RedisAI, launch an instance using docker:
```
Comment (lantiga) [inline docs/index.md]: docker -> Docker?

File: docs/index.md [hunk 3]
```diff
@@ -1 +1,75 @@
...
+## Start
+You will need a redis-server version 4.0.9 or greater. This should be
+available in most recent distributions:
+
+```sh
+redis-server --version
+Redis server v=4.0.9 sha=00000000:0 malloc=libc bits=64 build=c49f4faf7c3c647a
+```
+
+To start redis with the RedisAI module loaded, you need to make sure the dependencies can be found from redis.  One example on how to do this on Linux is:
```
Comment (lantiga) [inline docs/index.md]: found from redis -> found by Redis

**Comment by lantiga** [comment]
@K-Jo If you update I'll merge right away

**Comment by K-Jo** [comment]
@lantiga fixed

**Pull request merged**